query/common/src/main/java/org/codice/ddf/admin/common/fields/base/scalar/BaseHiddenField.java

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.admin.common.fields.base.scalar;
+
+import static org.codice.ddf.admin.api.fields.ScalarField.ScalarType.HIDDEN_STRING;
+import static org.codice.ddf.admin.common.report.message.DefaultMessages.emptyFieldError;
+
+import java.util.List;
+import java.util.Set;
+
+import org.codice.ddf.admin.api.report.ErrorMessage;
+import org.codice.ddf.admin.common.report.message.DefaultMessages;
+
+import com.google.common.collect.ImmutableSet;
+
+public class BaseHiddenField extends BaseScalarField<String> {
+
+    public static final String DEFAULT_FIELD_NAME  = "hidden password";
```

Comment:
I think we should change this to be "hidden". This currently assumes the implementation will always be a password.